mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/WEB-INF/src/com/liferay/todo/TodoPortlet.java

```diff
@@ -0,0 +1,62 @@
+package com.liferay.todo;
+
+import com.liferay.portal.kernel.servlet.SessionErrors;
+import com.liferay.portal.kernel.servlet.SessionMessages;
+import com.liferay.portal.kernel.util.ParamUtil;
+import com.liferay.portal.util.PortalUtil;
+import com.liferay.util.bridges.mvc.MVCPortlet;
+
+import javax.portlet.*;
+import java.io.IOException;
+
+/**
+ * Created by Mateo on 7/13/2017.
+ */
+public class TodoPortlet extends MVCPortlet {
+
+	public void doView(RenderRequest renderRequest, RenderResponse renderResponse)
+
+		throws IOException, PortletException {
+			System.out.println("Render view of Todo Portlet");
+
+			super.doView(renderRequest, renderResponse);
+	}
+
+	public void formAction(ActionRequest actionRequest, ActionResponse actionResponse)
```

Comment:
Method should be named by what is **does**, not what it **is**. For example, ```addTask()``` or ```deleteTask()```, not ```javaMethod()``` or ```formAction()``` .